src/node_buffer.cc

```diff
@@ -310,6 +316,71 @@ void Base64Slice(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+// bytesCopied = buffer.mask(mask, target, targetOffset, sourceStart, sourceEnd)
+void Mask(const FunctionCallbackInfo<Value> &args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  CopyDWord mask;
+  mask.as_dword = args[0]->Uint32Value();
```

Comment:
It doesn't hurt but you don't have to store the mask in a union, just a uint32_t is alright.